scripts/build.py

```diff
@@ -141,6 +165,8 @@ def get_cache_slug():
     CACHE_SLUG = get_cache_slug()
     BUILD_DIR = os.path.join('build', CACHE_SLUG)
 
+    # ensure_directory_exists trims file paths passed to it. Hence, directory
+    # paths require a trailing slash.
```

Comment:
Ah, I see. It's trimming in the sense that it removes everything after the last slash present in the path and sort of moves one level up.